index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
- [x] Great job 👍 . Kindly add your HTML / CSS linter configurations to your repossitory🙏 . This will help to ensure your code keeps to the standards of the microverse organization